### PR TITLE
Add 'document' as a component

### PR DESCRIPTION
**Pull request by quis**
## A component for linking to a downloadable file, or a list of such links

This is mostly a lift-and-shift from the Grails app, with some slightly more sensible naming of classes.

image

### Graceful degredation

IE6 | IE 8 | IE 9
--- | --- | ---
image | image | image

### CODE REVIEW

File: toolkit/scss/_document.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+%document-link,
+.document-link {
+  @include core-19;
+  margin: 5px 0;
+  padding: 10px 0 10px 0;
+  position: relative;
+  display: block;
+  width: 100%; // Trigger HasLayout on older IEs
```
Comment (tombye) [inline toolkit/scss/_document.scss]: I think we need these comments in the final CSS so maybe use `/* */` style?

File: toolkit/scss/_document.scss [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  display: block;
+  position: absolute;
+  top: 51%;
+  left: 0;
+  background: transparent;
+  text-transform: lowercase;
+  text-decoration: none;
+
+  &:before {
+    content: '.'; // Insert this with CSS so that it doesn't get read out
```
Comment (tombye) [inline toolkit/scss/_document.scss]: Same as the above for the style of comments.

File: toolkit/scss/_document.scss (hunk 1 above)
Comment (quis) [inline toolkit/scss/_document.scss]: Comments amended in 8b78170

**Comment by quis** [comment]
@tombye Made some changes for older versions of IE in 2cace49. See revised screenshots above.

**Pull request merged**